e2e/autoscript-apps/README.md

```diff
@@ -0,0 +1,55 @@
+<!--toc:start-->
+
+- [Autoscript Apps](#autoscript-apps)
+- [Running](#running)
+- [Adding an application](#adding-an-application)
+<!--toc:end-->
+
+# Autoscript Apps
+
+This is a package within the repository that contains miniature applications that
+represent a unit or piece of functionality. They exist on a singular multi-page
+web app which is bundled together via webpack.
+
+The point of this is to have an application which can serve various
+functionalities that are as bare bones as possible to ensure
+the functionality that is being supported can be tested.
+They contain no real ui, and are minimal in terms of what they do.
```

Comment:
I'd add the following: "Assertions (found in the `e2e/autoscript-suites/` directory) are made against `console.logs` or network requests recorded by Playwright."